packages/components/src/components/banner/banner.styles.ts

```diff
@@ -0,0 +1,89 @@
+import { css } from 'lit';
+
+const styles = css`
+  :host {
+    --mdc-banner-background-color: var(--mds-color-theme-background-solid-primary-normal);
+    --mdc-banner-border-color: var(--mds-color-theme-outline-secondary-normal);
+    --mdc-banner-text-color: var(--mds-color-theme-text-primary-normal);
+    --mdc-banner-icon-color: var(--mdc-banner-text-color);
```

Comment:
```suggestion
    --mdc-banner-icon-color: var(--mds-color-theme-text-primary-normal);
```

` --mdc-banner-text-color` is not used anywhere else in this file